Modules/_asynciomodule.c

```diff
@@ -2665,7 +2652,6 @@ static PyType_Slot Task_slots[] = {
     {Py_tp_clear, (inquiry)TaskObj_clear},
     {Py_tp_iter, (getiterfunc)future_new_iter},
     {Py_tp_methods, TaskType_methods},
-    {Py_tp_members, TaskType_members},
```

Comment:
is this `Py_tp_members` related to managed weakrefs?

Reply:
That's how `__weaklistoffset__` was set earlier.